Mathlib/Data/Matrix/Basis.lean

```diff
@@ -4,7 +4,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jalex Stark, Kim Morrison, Eric Wieser, Oliver Nash, Wen Yang
 -/
 import Mathlib.Data.Matrix.Basic
-import Mathlib.LinearAlgebra.Matrix.Trace
```

Comment:
Should this become an `#assert_not_imported` or something like that?

Reply:
I'll add an `assert_not_exists` for `Matrix.trace`.